### PR TITLE
update README and add new screenshots

### PR DESCRIPTION
**Pull request by vrothberg**
Update the README with instructions on how to install and use the extension along with some fresh screenshots.

### CODE REVIEW

**Comment by vrothberg** [comment]
@jeffmaury @dgolovin @benoitf PTAL

**Review by benoitf — APPROVED** [review]
I reported few suggestion glitches

FYI README content is usually included in the catalog and then it's being displayed when you open special links to podman desktop

using the shortcut `image` won't work as it's relative links

you should update it with links starting with `https://raw.githubusercontent.com/redhat-developer/podman-desktop-redhat-account-ext/main/images/...`

**Comment by vrothberg** [comment]
> you should update it with links starting with `https://raw.githubusercontent.com/redhat-developer/podman-desktop-redhat-account-ext/main/images/...`

Thanks!  Can we do that in a follow-up PR?  Otherwise, they won't show up now for review as the images are not yet uploaded/pushed.  Or is there a way to do that?

**Comment by benoitf** [comment]
follow-up works for me, I just wanted to bring awareness and approved the PR

**Comment by vrothberg** [comment]
Thanks for the review, @benoitf !

**Comment by vrothberg** [comment]
@rhatdan FYI

File: README.md [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
-# Red Hat Account extension
+An extension for Podman Desktop to simplify logging into and creating a Red Hat account.  The extension opens ssh.redhat.com in the browser to retrieve an SSO token upon successful login.  The SSO token is then used to log into the [registry.redhat.io](https://catalog.redhat.com/) Container Registry and to register the Linux virtual machine powering Podman Desktop via subscription-manager to grant the containers access to protected Red Hat content such as RHEL repositories.
 
-This extension plugs into Podman Desktop an authentication provider that allows login to Red Hat SSO
+# Installation
 
-# Run and build
+The extension is currently available in an Alpha version but ready to test.  Extension for Podman Desktop are shipped as OCI container images.  Please refer to the [Podman Desktop documentation](https://podman-desktop.io/docs/extensions/install) for installation instructions and to [Quay.io](https://quay.io/repository/redhat-developer/podman-desktop-redhat-account-ext?tab=tags) for available tags.
 
+The first available Alpha image is `quay.io/redhat-developer/podman-desktop-redhat-account-ext:0.0.1-alpha.1`.
```
[the remark this answers is not in the document]
Comment (dgolovin) [reply README.md:9]: I will change it to 0.0.2-alpha.1

[the remark this answers is not in the document]
Comment (dgolovin) [reply README.md:9]: In a follow up PR

**Comment by dgolovin** [comment]
I'll take care of images in follow up PR as well

**Review by dgolovin — APPROVED** [review]

**Pull request merged**